GoodMedicalApp.BusinessServices/Services/Implementation/OperationService.cs

```diff
@@ -63,6 +65,58 @@ public Operation GetItemById(int id)
             }
         }
 
+        public OperationTransfer GetTransferItemById(int id)
+        {
+            var operation = GetItemById(id);
+            if (operation != null)
+            {
+                var currentMedicines = new List<int>();
+                if (operation.Medicines != null && operation.Medicines.Any())
+                {
+                    currentMedicines = operation.Medicines.Select(x => x.Id).ToList();
+                }
+
+                var operationTransfer = new OperationTransfer()
+                {
+                    Id = operation.Id,
+                    CurrentTypeOperation = operation.CurrentTypeOperation,
+                    TreatmentId = operation.TreatmentId,
+                    Medicines = currentMedicines
+                };
+
+                return operationTransfer;
+            }
+            else
```

Comment:
Try not to return null. Signal about error or missing data in some other way

Reply:
Fixed. Added throwing exception